### PR TITLE
update and delete helper functions

### PR DESCRIPTION
**Pull request by tommyconner96**
added helper functions for update and delete workOrders. These can be used for PUT and DELETE routes.

### CODE REVIEW

**Review by germainep — CHANGES_REQUESTED** [review]
Can you re-run eslint fix and resubmit this.

File: api/workOrder/woModel.js [hunk 1]
```diff
@@ -16,9 +16,24 @@ const create = async (wo) => {
   return db('workOrders').insert(wo).returning('*');
 };
 
+const update = (id, wo) => {
+  console.log(wo);
+  return db('workOrders')
+    .where({ id: id })
```
Comment (germainep) [inline api/workOrder/woModel.js]: here and on line 12 the id column in the database is named "requestId" so it should be ```.where({requestId: id})```

Comment (tommyconner96) [reply api/workOrder/woModel.js]: you're right. fixed, thank you

**Comment by tommyconner96** [comment]
> Can you re-run eslint fix and resubmit this.

Done!

File: api/workOrder/woModel.js (hunk 1 above)
Comment (germainep) [reply api/workOrder/woModel.js]: actually the migration is wrong there is both an Id column and a requestId column which do you guys think we should keep?

Comment (tommyconner96) [reply api/workOrder/woModel.js]: probably just ID would be simpler. Yeah, I don't think we need both. I'm running into more linting problems I will switch it back to being ID, run prettier again and then hopefully we can get this to pass all the build tests again

Comment (germainep) [reply api/workOrder/woModel.js]: I'm going to go ahead and change the schema to use ID then and get that pull-request in

**Review by germainep — APPROVED** [review]

**Review by sophiasagan — APPROVED** [review]
Helper functions are done really well.

**Pull request merged**